lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -194,7 +193,7 @@ def _handle_error(self, ev):
         self._msg_hub.handle_error(ev)
 
     # pylint: disable=protected-access,unused-argument
-    def _get_rule_match_flow_msgs(self, imsi, _, __, ip_addr, ambr, rule, version):
+    def _get_rule_match_flow_msgs(self, imsi, _, __, ip_addr, ambr, rule, version, shard_id):
```

Comment:
```suggestion
    def _get_rule_match_flow_msgs(self, imsi, _, _, ip_addr, ambr, rule, version, shard_id):
```

Reply:
It actually has to be double underscore since it's a duplicate parameter